index.html

```diff
@@ -399,70 +399,66 @@ <h2>Legal Effects</h2>
-          for data sales and targeted advertising, including through an “authorized agent by way of,
-          among other things, a technology, including, but not limited to, an Internet link or a
-          browser setting, browser extension or global device setting.” Under the CDPA, the GPC signal
-          will be intended to communicate a request to opt out of both the sale of their personal
-          information and the use of their personal information for targeted advertising.
-        </li>
-      </ul>
+      <h3>Calfornia Consumer Privacy Act (CCPA)</h3>
       <p>
-        GPC could potentially be used to indicate rights in other jurisdictions as well. For example:
+        Under the CCPA, the GPC signal will be intended to communicate a Do Not Sell request
+        from a global privacy control, as per [[?CCPA-REGULATIONS]] §999.315 for that browser or
+        device, or, if known, the consumer.
+      </p>
+      <p>
+        Where the GPC signal conflicts with the existing privacy settings a consumer has with
+        the business, the business shall respect the GPC signal but may notify the consumer of
+        the conflict and give the consumer an opportunity to confirm the business-specific
+        privacy setting or participation in the financial incentive program [[?CCPA-REGULATIONS]]
+        §999.315(c)(2).
+      </p>
+      <h3>Colorado Privacy Act (CPA)</h3>
+      <p>
+        The CPA gives consumers the legal right to opt out of both the sale of their information 
+        as well as the use of their data for cross-site targeted advertising, including through 
+        the use of “universal opt-out mechanisms that clearly communicate a consumer’s affirmative, 
+        freely given, and unambiguous choice to opt out.” Under the CPA, the GPC signal will be 
+        intended to communicate a request to opt out of both the sale of their personal information
+        and the use of their personal information for targeted advertising.
+      </p>
+      <h3>Connecticut Data Privacy Act (CDPA)</h3>
+      <p>
+        Similarly, the CDPA gives consumers separate opt-out rights for data sales and targeted 
+        advertising, including through an “authorized agent by way of, among other things, a 
+        technology, including, but not limited to, an Internet link or a browser setting, browser 
+        extension or global device setting.” Under the CDPA, the GPC signal will be intended to 
+        communicate a request to opt out of both the sale of their personal information and the 
+        use of their personal information for targeted advertising.
+      </p>
+      <h3>Nevada Revised Statutes Chapter 603A (NRS 603A)</h3>
+      <p>
+        Under NRS 603A, a GPC signal will be intended to communicate a Do Not Sell My Personal
+        Information request [[?SB220]].
+      </p>
+      <h3>General Data Protection Regulation (GDPR)</h3>
```

Comment:
```suggestion
      <h3>European General Data Protection Regulation (GDPR)</h3>
```

I think? Or "EU" or similar?

Reply:
Thank you, @jyasskin. I added "EU."